creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -46,6 +46,10 @@ const healthCheckVerifySignature = (req, res, next) => {
  * `healthCheckComponentService`.
  */
 const healthCheckController = async (req) => {
+  if (config.get('isReadOnlyMode') === true) {
+    return errorResponseBadRequest()
```

Comment:
super nit but these should all be errorResponseServerError. errorResponseBadRequest (400 response code) means there's a issue with the client request

Reply:
Resolved, thanks!